### DISCUSSION TITLE
1.8.9webdav服务总是自动关闭

### DISCUSSION THREAD

**Discussion by LKorint** [messense/aliyundrive-webdav#547] [Q&A] [answered]
### 问题描述

路由器是小米ac2100，openwrt系统。1.8.9的webdav插件在播放视频时总是自动变成未运行。

### 重现步骤

1、安装1.8.9的ipk
2、使用infuse连接webdav
3、观看视频不到一分钟，openwrt的插件变为未运行，无法连接

### 版本

1.8.9

### 运行平台

openwrt

### refresh token 获取方式

扫码

### 日志

抱歉，开启调试模式，并未发现日志有报错，只是自动停止运行，只能手动再次开启。

**Comment by prunes81485216** [comment, marked as answer]
应该是路由器内存不够引起的问题

**Reply by LKorint** [reply]
检查了一遍，进程瞬时占用内存过高被oom killer杀死了，在ssh配置一下进程的score，防杀进程就解决了，感谢提供思路
